src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -73,7 +77,8 @@ public async Task ExecuteAsync(string saveCdkDirectoryPath, string projectDispla
             {
                 if (newDirectoryCreated)
                     _directoryManager.Delete(saveCdkDirectoryPath);
-                throw new InvalidSaveDirectoryForCdkProject(errorMessage);
+                errorMessage = "Failed to generate deployment project." + Environment.NewLine + errorMessage;
```

Comment:
Use string interpolation instead of concatenating strings.

Reply:
Fixed